nubis/bin/example-consul2unixusers.conf

```diff
@@ -0,0 +1,5 @@
+[consul]
+server = hostname.for.your.consul.server
+scheme = http
+port = 80
+path = path/to/key/where/consul2ldap/wrote/ldap/CNs
```

Comment:
Looks like this doesn't need a starting '/', I find it confusing. I'd at least make sure to check for that. But ideally, I'd support it.